rstudio/rstudio.sh

```diff
@@ -80,29 +75,34 @@ if [[ "${ROLE}" == 'Master' ]]; then
     echo "RStudio user name and password must not be the same."
     exit 3
   fi
+
   # Install RStudio Server
-  update_apt_get
-  apt-get install -y software-properties-common
   if [[ "${OS_ID}" == "ubuntu" ]]; then
-    run_with_retries apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E298A3A825C0D65DFD57CBB651716619E084DAB9
-    add-apt-repository "deb http://cran.r-project.org/bin/linux/ubuntu ${OS_CODE}-cran35/"
-    rstudio_server_package=rstudio-server-${RSTUDIO_VERSION}-amd64.deb
+    REPOSITORY_KEY=E298A3A825C0D65DFD57CBB651716619E084DAB9
   else
-    run_with_retries get_apt_key_for_debian
-    add-apt-repository "deb http://cran.r-project.org/bin/linux/debian ${OS_CODE}-cran35/"
-    rstudio_server_package=rstudio-server-${OS_CODE}-${RSTUDIO_VERSION}-amd64.deb
+    REPOSITORY_KEY=E19F5F87128899B192B1A2C2AD5F960A256A04AF
   fi
+  run_with_retries apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys ${REPOSITORY_KEY}
+  apt-get install -y software-properties-common
+  add-apt-repository "deb http://cran.r-project.org/bin/linux/${OS_ID} ${OS_CODE}-cran35/"
   update_apt_get
-  apt install -y r-base r-base-dev gdebi-core
+  apt-get install -y r-base r-base-dev gdebi-core
 
+  # Download and install RStudio Server package:
+  # https://rstudio.com/products/rstudio/download-server/debian-ubuntu/
+  if [[ ${OS_CODE} == stretch ]]; then
```

Comment:
Why there is no check for Debian 10?

Reply:
Because Debian 10 and Ubuntu 18 have the same packages:
https://rstudio.com/products/rstudio/download-server/debian-ubuntu/